hail/python/test/hail/methods/test_impex.py

```diff
@@ -442,6 +442,120 @@ def test_missing_float_entries(self):
         assert gl_gp == [hl.Struct(GL=[None, None, None], GP=[0.22, 0.5, 0.27]),
                          hl.Struct(GL=[None, None, None], GP=[None, None, None])]
 
+    def test_same_bgzip(self):
+        mt = hl.import_vcf(resource('sample.vcf'))
+        f = new_temp_file(suffix='vcf.bgz')
+        hl.export_vcf(mt, f)
+        assert hl.import_vcf(f)._same(mt)
+
+    def test_sorted(self):
+        mt = hl.utils.range_matrix_table(10, 10).filter_cols(False)
```

Comment:
Do you want the matrix table to have multiple partitions?

Reply:
yeah, good catch